tools/msys2_fetch_and_build_all_tx16s.sh

```diff
@@ -1,12 +1,38 @@
 #! /usr/bin/env bash
 
-## Bash script to show how to get EdgeTX source from GitHub (main branch),
+## Bash script to show how to get EdgeTX source from GitHub,
 ## how to build TX16S firmware, Companion, Simulator, radio simulator
 ## library and how to create an installation package.
 ## Let it run as normal user in MSYS2 MinGW 64-bit console (blue icon).
 
-PAUSEAFTEREACHLINE="false"
-STEP=1
+# -----------------------------------------------------------------------------
+export BRANCH_NAME="main"  # main|2.8|2.9|...
```

Comment:
You might want to explicitly opt to limit to 2.8 and newer branches/tags only, as some build commands are to be handled differently for older firmware versions.

Reply:
commented as 2.8 and above